### DISCUSSION TITLE
Which rule changes multiline array inside function argument into single line?

### DISCUSSION THREAD

**Discussion by christhofer** [PHP-CS-Fixer/PHP-CS-Fixer#7185] [Q&A] [answered]
My .php-cs-fixer.php
```php
<?php

return (new PhpCsFixer\Config())
    ->setRules([
        '@PSR12' => true,
        '@Symfony' => true,
    ])
    ->setLineEnding("\n");
```

It formats my code like this.

```diff
- throw \Illuminate\Validation\ValidationException::withMessages([
-     'attr1' => __('invalid', ['attribute' => 'Attribute 1']),
-     'attr2' => __('invalid', ['attribute' => 'Attribute 2']),
- ]);
+ throw \Illuminate\Validation\ValidationException::withMessages(['attr1' => __('invalid', ['attribute' => 'Attribute 1']), 'attr2' => __('invalid', ['attribute' => 'Attribute 2'])]);
```

It is not triggered if I disable the `@Symfony` set.

Which rule converts the code above? And can we enforce the style to prefer the former?

I have looked around the [Symfony rules](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/doc/ruleSets/Symfony.rst), but can't find it.

**Comment by Wirone** [comment]
Please run Fixer with `--verbose --diff --dry-run` and there will be displayed list of fixers that would be applied. I believe you can find rule which is changing this by its name, if you're not sure you, can check by adding `--rules=<name>`.

**Reply by christhofer** [reply]
Thank you for the quick response.
The format run automatically on save using VS Code junstyle.php-cs-fixer extension.
I only find the Diff command and can see the difference, but can't find a way to use --verbose and --dry-run

I'll try to do format with command line.

Is there an online playground to try the rules and see the verbose dry-run output there?

**Reply by christhofer** [reply, marked as answer]
I finally get to do it from a terminal.

It is showing:
```
1) app\Http\Controllers\MyController.php (single_line_throw, trim_array_spaces)
```

**Reply by Wirone** [reply]
> Is there an online playground to try the rules and see the verbose dry-run output there?

No, we unfortunately don't have playground yet.
